Documentation/Contributors/CodingGuide/README.md

```diff
@@ -145,6 +152,14 @@ try {
 /*eslint-enable no-empty*/
 ```
 
+* If possible, use `//eslint-disable-line` to disable specific rules for one line.
```

Comment:
I would put this before the block scope version and instead of `if possible` say something like "When disabling linting for a single line you... and "when disabling linting for a block of code"